compiler/src/iree/compiler/PluginAPI/PluginManager.cpp

```diff
@@ -105,25 +105,36 @@ LogicalResult PluginManagerSession::activatePlugins(MLIRContext *context) {
   // sorting accordingly. For now, what you say is what you get.
   for (auto &pluginId : options.plugins) {
     if (options.printPluginInfo) {
-      llvm::errs() << "[IREE plugins]: Activating plugin '" << pluginId
+      llvm::errs() << "[IREE plugins]: Initializing plugin '" << pluginId
                    << "'\n";
     }
     auto foundIt = allPluginSessions.find(pluginId);
     if (foundIt == allPluginSessions.end()) {
-      auto diag = mlir::emitError(mlir::UnknownLoc::get(context))
-                  << "could not activate requested IREE plugin '" << pluginId
-                  << "' because it is not registered (available plugins: ";
-      llvm::interleaveComma(getAvailableIds(), diag);
-      diag << ")";
+      llvm::errs()
```

Comment:
OOC do we have a test for this?

Reply:
I need to come back through and add tests for all of this. But need to get past a chicken-egg issue of how to manage the tests and configurations on the CI... trust me, not going to leave this untested.